File: leetcode/2025-12/2025-12-25-twoSum/python/solution.py
from typing import List

list = [2, 7, 11, 15]


def two_sum(list: List[int], target: int) -> List[int]:
    list_len = len(list)
    for i in range(list_len):
        for j in range(list_len):
            if list[i] + list[j] == target and i != j:
                return [i, j]
    return []

File: leetcode/2025-12/2025-12-25-twoSum/go/solution.go
func twoSum(nums []int, target int) []int {
	for i := 0; i < len(nums); i++ {
		for j := 0; j < len(nums); j++ {
			if sum := nums[i] + nums[j]; sum == target && i != j {
				return []int{i, j}
			}
		}
	}
	return nil
}
